js/apps/account-ui/src/root/Root.tsx

```diff
@@ -13,11 +13,28 @@ import { environment } from "../environment";
 import { keycloak } from "../keycloak";
 import { joinPath } from "../utils/joinPath";
 import { PageNav } from "./PageNav";
+import { ExternalLinkAltIcon } from "@patternfly/react-icons";
 
 import style from "./Root.module.css";
 
+const ReferrerLink = () => {
+  const { t } = useTranslation();
+  const searchParams = new URLSearchParams(location.search);
+
+  return searchParams.has("referrer_uri") ? (
+    <a
```

Comment:
I think this could be a PatternFly component instead, if you combine the [link variants](https://www.patternfly.org/v4/components/button#variant-examples) and [link as buttons](https://www.patternfly.org/v4/components/button#links-as-buttons) examples of the button component.

Something like:
```tsx
<Button component="a" href={href} variant="link" icon={<ExternalLinkSquareAltIcon />} iconPosition="right" isInline>
  {t("backTo", { app: searchParams.get("referrer") })}
</Button>
```